liberty-maven-plugin/src/main/resources/net/wasdev/wlp/maven/plugins/MvnMessages.properties

```diff
@@ -331,5 +331,5 @@ warn.install.license.explanation=The license file wlp/lafiles/LI_en is not found
 warn.install.license.useraction=Download and reinstall the Liberty license jar file to your local repository or internal custom repository.
 
 error.install.license=CWWKM2183E: Failed to install the Liberty production license {0}. Return code is {1}.
-error.install.license.explanation=Failed to install the Liberty production license to the runtime installation.
-error.install.license.useraction=Check the return code and execution failure log to investigate the cause of the failure.
+error.install.license.explanation=Failed to install the Liberty production license to the runtime installation. The current installation might contain features that are not supported by the production license terms and conditions.
+error.install.license.useraction=Check the return code and execution failure log to investigate the cause of the failure. If the current installation contains features which are not supported by the Liberty production license terms and conditions, follows the instruction in the execution failure log to uninstall these features and try again.
```

Comment:
or should we also recommend they obtain the license they need to cover the features they are using?